wordpress-mobile/WordPress-iOS
# Restore support for animated GIFs in Gutenberg.

When we implemented our custom ImageLoader for RN, to support private sites, we accidentally removed support for playing animated GIFs while inside GB editor.
This PR changes the loading system of images to use the `RCTAnimated` class when a GIF is being downloaded.

One of the side effects of that change is that if the post featured image on the post/page is an animated GIF we show the animation.

<img src="https://user-images.githubusercontent.com/651601/83644137-da132d00-a5a8-11ea-8b59-5dd32e958808.gif" width=300>

Fixes #

To test:
 - Start the app
 - Create a new Post using GB
 - Add an image block and configure it to use an animated GIF
 - See if you can see the GIF animation on the editor
 - Publish the post
 - Go to the list of posts
 - See if the post card in the list of posts shows the animated GIF
 - Test this with WP.com sites, WP.org sites, Atomic sites, and WP.com private sites.

PR submission checklist:
